cartoframes/examples.py

```diff
@@ -0,0 +1,381 @@
+"""Download, preview, and query example datasets for use in cartoframes
```

Comment:
@andy-esch I love your point about not even needing a CARTO account to try it, I'd write it down here, and maybe in the [landing features list](http://cartoframes.readthedocs.io/en/latest/#features) with a link.

Reply:
Great suggestion, added to both :)